modules/ROOT/pages/index.adoc

```diff
@@ -13,8 +13,9 @@ They are typically transmitted over transports (such as HTTP(s), SFTP, and excha
 All of this is integrated into the Mule runtime.
 
 Use Anypoint Partner Manager to set up, manage, and monitor the flow of messages between you and your business partners.
-Create message flows, the paths along which a message travels from its source, through a transformation map, to its target.
-Monitor each transmission, the transfer of a payload and the validation of the messages it contains. The ability to track and report on this drives greater operational efficiency while managing B2B transactions.
+Create message flows (the paths along which a message travels from its source) through a transformation map, to its target.
```

Comment:
```suggestion
Create message flows. A message flow is the path along which a message travels from its source, through a transformation map, to its target.
```

Reply:
This was edited by Kim and if Amit didn't change it, I"m inclined to leave it.

Reply:
The original was "Create message flows, the paths along which a message travels from its source, through a transformation map, to its target." That sounds to me like the definition of a message flow is "the path along which a message travels from its source, through a transformation map, to its target". I think "Create message flows (the paths along which a message travels from its source) through a transformation map, to its target." garbles the definition incorrectly, so I'd recommend rewording this.